src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -109,8 +109,8 @@ define(['globalize', 'dom', 'emby-checkbox', 'emby-select', 'emby-input'], funct
         html += '<div class="metadataFetcher" data-type="' + availableTypeOptions.Type + '">';
         html += '<h3 class="checkboxListLabel">' + globalize.translate('LabelTypeMetadataDownloaders', globalize.translate(availableTypeOptions.Type)) + '</h3>';
         html += '<div class="checkboxList paperList checkboxList-paperList">';
-        for (var i = 0; i < plugins.length; i++) {
-            var plugin = plugins[i];
+
+        plugins.array.forEach((plugin, index) => {
```

Comment:
I have an error `TypeError: Cannot read property 'forEach' of undefined`.
```suggestion
        plugins.forEach((plugin, index) => {
```